src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/ShardSyncerTest.java

```diff
@@ -955,12 +967,14 @@ public final void testDetermineNewLeasesToCreateSplitMergeHorizon1() {
     /**
      * Test CheckIfDescendantAndAddNewLeasesForAncestors (initial position TrimHorizon)
      * Shard structure (each level depicts a stream segment):
-     * 0 1 2 3 4 5- shards till epoch 102
-     * \ / \ / | |
-     * 6 7 4 5- shards from epoch 103 - 205
-     * \ / | /\
-     * 8 4 9 10 - shards from epoch 206 (open - no ending sequenceNumber)
+     * 0 1 2 3 4   5- shards till epoch 102
+     * \ / \ / |   |
+     *  6   7  4   5- shards from epoch 103 - 205
+     *   \ /   |  /\
+     *    8    4 9  10 - shards from epoch 206 (open - no ending sequenceNumber)
      * Current leases: (4, 5, 7)
+     * Initial position: TRIM_HORIZON
+     * Leases to create: (6)
```

Comment:
We should be creating (0, 1) from TRIM. Looks like it's correct in the actual test.

Reply:
Copy pasted badly, updated this javadoc.